specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json

```diff
@@ -0,0 +1,1095 @@
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/PrivateLinkScopesResource"
+        }
+      ],
+      "description": "An Azure Arc PrivateLinkScope definition."
+    },
+    "PrivateLinkScopeValidationDetails": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "publicNetworkAccess": {
+          "description": "Indicates whether machines associated with the private link scope can also use public Azure Arc service endpoints.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "connectionDetails": {
+          "type": "array",
+          "description": "List of Private Endpoint Connection details.",
+          "items": {
+            "$ref": "#/definitions/ConnectionDetail"
+          }
+        }
+      }
+    },
+    "ConnectionDetail": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "privateIpAddress": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The private endpoint connection private ip address"
+        },
+        "linkIdentifier": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The private endpoint connection link identifier"
+        },
+        "groupId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The private endpoint connection group id"
+        },
+        "memberName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The private endpoint connection member name"
+        }
+      }
+    },
+    "KubernetesConfigurationPrivateLinkScopeProperties": {
+      "type": "object",
+      "description": "Properties that define a Azure Arc PrivateLinkScope resource.",
+      "required":["clusterArmId"],
+      "properties": {
+        "publicNetworkAccess": {
+          "description": "Indicates whether machines associated with the private link scope can also use public Azure Arc service endpoints.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Current state of this PrivateLinkScope: whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Provisioning ,Succeeded, Canceled and Failed.",
+          "readOnly": true
+        },
+        "clusterArmId": {
```

Comment:
>clusterArmId [](http://example.com/codeflow?start=9&length=12)

Don't include abbreviations in property names. Consider using clusterResourceId for this property.